index.js

```diff
@@ -72,20 +73,24 @@ module.exports = function(config) {
     mailserver.setAutoRelayMode(true, config.autoRelayRules);
   }
 
-  // Default to run on same IP as smtp
-  var webIp = config.webIp ? config.webIp : config.ip;
-  web.start(config.web, webIp, mailserver, config.webUser, config.webPass, config.basePathname);
+  if (!config.noWeb) {
+    // Default to run on same IP as smtp
+    var webIp = config.webIp ? config.webIp : config.ip;
+    web.start(config.web, webIp, mailserver, config.webUser, config.webPass, config.basePathname);
 
-  if (config.open){
-    var open = require('open');
-    open('http://' + (config.ip === '0.0.0.0' ? 'localhost' : config.ip) + ':' + config.web);
+    if (config.open){
+      var open = require('open');
+      open('http://' + (config.ip === '0.0.0.0' ? 'localhost' : config.ip) + ':' + config.web);
+    }
   }
 
   process.on('SIGTERM', function () {
-    async.parallel([
-      mailserver.end,
-      web.close
-    ], function(err, results) {
+    var tasks = [mailserver.end];
+    if(web) {
```

Comment:
I think this will always be truthy as it's the module's export object. It could be useful to have a method like `web.isListening()`?

Now that I think of it, [the `createServer` call in web.js](https://github.com/djfarrelly/MailDev/blob/77392aceecd8d4edf2ae37c997909bade073b3ca/lib/web.js#L8) should likely be moved into the `start` function then the `isListening()` function can check if `server` is defined or not. If you'd like to make this change, that works with me or I can hack it together after this is merged. Let me know what you think!